client/config/toml.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import (
+	"bytes"
+	"os"
+	"text/template"
+
+	"github.com/spf13/viper"
+	tmos "github.com/tendermint/tendermint/libs/os"
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+)
+
+const defaultConfigTemplate = `# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
+
+
+chain-id = "{{ .ChainID }}"
+keyring-backend = "{{ .KeyringBackend }}"
+output = "{{ .Output }}"
+node = "{{ .Node }}"
+broadcast-mode = "{{ .BroadcastMode }}"
+`
+
+// initConfigTemplate initiates config template that will be used in
+// writeConfigFile
+func initConfigTemplate() (*template.Template, error) {
+	tmpl := template.New("clientConfigFileTemplate")
+	configTemplate, err := tmpl.Parse(defaultConfigTemplate)
+	if err != nil {
+		return nil, err
+	}
+
+	return configTemplate, nil
+}
+
+// writeConfigFile renders config using the template and writes it to
+// configFilePath.
+func writeConfigFile(configFilePath string, config *ClientConfig, configTemplate *template.Template) error {
+	var buffer bytes.Buffer
+
+	if err := configTemplate.Execute(&buffer, config); err != nil {
+		return err
+	}
+
+	tmos.MustWriteFile(configFilePath, buffer.Bytes(), 0644)
+	return nil
+
```

Comment:
`MustWriteFile` is removed from future tendermint releases. I suggest adopting the following (and equivalent) form:

```suggestion
	return ioutil.WriteFile(configFilePath, buffer.Bytes(), 0644)
```

Reply:
Done!